angular-ngc/tools/BUILD.bazel

```diff
@@ -34,6 +35,19 @@ ts_project(
 exports_files(
     [
         "karma.conf.js",
+        "test-setup.ts",
     ],
     visibility = ["//visibility:public"],
 )
+
+esbuild(
+    name = "test_bootstrap",
+    testonly = 1,
+    entry_point = "//tools:test-bootstrap.js",
```

Comment:
This could just be `:test-bootstrap.js` 🤷

Reply:
Yes, you are right, updated.